com.budgetwithbuckets.Buckets.desktop

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=Buckets
-Exec=com.budgetwithbuckets.Buckets
+Exec=buckets.sh
 Type=Application
-Icon=com.budgetwithbuckets.Buckets
+Icon=../icons/hicolor/512x512/apps/com.budgetwithbuckets.Buckets.png
```

Comment:
this was correct. The change makes it INCORRECT.

Reply:
1. it's never a path
2. it never has an extension.

Reply:
While building with it as it was, it complained with this warning:
<img width="420" alt="Screen Shot 2023-09-28 at 11 28 57 AM" src="https://github.com/flathub/flathub/assets/161727/80a33702-ac01-42ba-8dd0-a972007b9a16">

But I can change it back.

Reply:
Changed back